stock_by_warehouse_stock/18n/es.po

```diff
@@ -0,0 +1,61 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* stock_by_warehouse_stock
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-06-01 21:05+0000\n"
+"PO-Revision-Date: 2020-06-01 21:05+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: stock_by_warehouse_stock
+#: model:ir.model,name:stock_by_warehouse_stock.model_stock_move_line
+msgid "Product Moves (Stock Move Line)"
+msgstr "Movimientos de Producto (Stock Move Line)"
+
+#. module: stock_by_warehouse_stock
+#: model_terms:ir.ui.view,arch_db:stock_by_warehouse_stock.view_stock_move_operations
+msgid "Stock Availability from the warehouse:"
+msgstr "Disponibilidad de existencias del almacén:"
+
+#. module: stock_by_warehouse_stock
+#: model:ir.model,name:stock_by_warehouse_stock.model_stock_move
+msgid "Stock Move"
+msgstr "Movimiento de existencias"
+
+#. module: stock_by_warehouse_stock
+#: model:ir.model.fields,field_description:stock_by_warehouse_stock.field_stock_move__stock_picking_outgoing
+msgid "Stock Picking Outgoing"
```

Comment:
Where this term come from?

I didn't find it anywhere, and it should be "picking" -> "transfer"

Reply:
@CarmenMiranda can you fuly regenerate the PO files please?

Reply:
@luisg123v 
You can find it [here](https://github.com/Vauxoo/addons-vauxoo/pull/1415/commits/ec4d92929ab4c4fa48268fba31d9edcbd2314a5f?file-filters%5B%5D=.png&file-filters%5B%5D=.po&file-filters%5B%5D=.py&file-filters%5B%5D=.xml#diff-a9a9d958fc5ad825bee0d792d534c94fR16) Is just a field that I'm using to hide the widget on the `stock.move` where its `stock.picking` is not an outgoing one.
The field is on the view but is invisible

Reply:
OK